package.json

```diff
@@ -125,7 +125,8 @@
     "release": "bumpp package.json --commit --push --tag && git update-ref refs/heads/release refs/heads/main && git push origin release",
     "compile": "tsup ./src/extension.ts --external vscode --minify",
     "watch": "tsup ./src/extension.ts --external vscode --watch --sourcemap",
-    "test": "vitest run --config vite.global-config.ts",
+    "test": "vscode-test",
+    "test:watch": "vscode-test --watch-files src/**/*.ts --watch-files test/**/*.test.tsx --watch-files test/**/*.test.tsx",
```

Comment:
1. Do we have any tsx test files?
2. Why is the second pattern repeated?

Reply:
1. Good question 😏 
2. Good question 😏